setup.php

```diff
@@ -110,7 +110,8 @@ function plugin_init_example() {
    $PLUGIN_HOOKS['item_empty']['example']      = array('Computer' => 'plugin_item_empty_example');
 
    // Restrict right
-   $PLUGIN_HOOKS['item_can']['example']        = ['Computer' => ['PluginExampleComputer', 'restrict']];
+   $PLUGIN_HOOKS['item_can']['example']          = ['Computer' => ['PluginExampleComputer', 'item_can']];
```

Comment:
Hum... I guess https://github.com/pluginsGLPI/example/pull/16 and this one has to be merged; right?